### ISSUE TITLE
[Feature Request]:  Remapping the controller buttons directly from the application

### ISSUE THREAD

**Issue by Bobillho** [Paliverse/DualSenseX#191]
• The most used function, besides haptics, triggers and ambient color. At least I think that **_remapping_** directly from the application would be used by most users. 

At the moment there is already this aspect integrated in the steam layout if you have activated the support, but with a much easier to use interface and directly from **_DSX_**, the life of users would be much easier and more pleasant.
Every player wants to make his gameplay as pleasant as possible for him individually, not to follow a default series. This remapping function allows you to do exactly as you like.

It could be added very simply as an extension of the _Sense2Key_ function, to have a much friendlier interface to change the buttons.

**Comment by Delusionzzz** [comment]
I agree, me personally I would love to be able to remap my buttons directly from the app. I am able to do this for my 2 back paddles and the two "Fn" keys on my dualsense edge but not any of the other buttons on the controller, and its kinda disappointing.
